src/livekit/LivekitTransport.ts

```diff
@@ -86,6 +88,7 @@ export class LivekitTransport {
   async connect(): Promise<void> {
     await this.room.connect(this.url, this.token, { autoSubscribe: true })
     this.logger.log(`Connected to livekit room ${this.room.name}`)
+    if (this.onRoomConnected) this.onRoomConnected(this.room)
```

Comment:
one small thing, is this enough or do you want to listen to the connect event in the room? That would handle re-connecting as well.

Reply:
> one small thing, is this enough or do you want to listen to the connect event in the room? That would handle re-connecting as well.

Yes, the idea is to create the LiveKit VoiceChat from that Room. If the Room changes, it will start over again.

I will merge this after some testing.